packages/starboard-components/rollup.config.js

```diff
@@ -0,0 +1,24 @@
+import copy from 'rollup-plugin-copy'
+import modify from 'rollup-plugin-modify'
+
+export default {
+	// input: 'css/components/button/button-config.css',
+	// output: {
+	// 	file: '../../src/css/components/button.css',
+	// 	format: 'css'
+	// },
+  plugins: [
+    copy({
+      targets: [
+        { src: 'css/components/button/button-config.css', dest: '../../src/css/components' },
+        { src: 'liquid/components/button/button.liquid', dest: '../../shopify/snippets' }
+        // { src: ['assets/fonts/arial.woff', 'assets/fonts/arial.woff2'], dest: 'dist/public/fonts' },
+        // { src: 'assets/images/**/*', dest: 'dist/public/images' }
+      ]
+    }),
+    modify({
+      find: '@import "tailwindcss/components";',
+      replace: '@import "tailwindcss/components";\n/** Anchour Components */\nimport "@anchour/starboard/css/components/button/index";\nimport "components/button";'
+    })
+  ]
+};
```

Comment:
Going to see if rollup works right now, but I am learning the fs stuff in Node, so that may be a better route long term. I may try to convert to that this week.